subprojects/core/src/main/java/org/gradle/initialization/DefaultGradleLauncher.java

```diff
@@ -195,8 +199,8 @@ private void prepareSettings() {
 
     private void prepareProjects() {
         if (stage == Stage.LoadSettings) {
-            projectsPreparer.prepareProjects(gradle);
-
+            ClassLoaderScope projectClassLoaderScope = buildSourceBuilder.buildAndCreateClassLoader(gradle.getSettings().getRootDir(), gradle.getStartParameter());
```

Comment:
❓ isn't that scope the `buildSrcClassLoaderScope` instead of `projectClassLoaderScope`? It's ID is the absolute path of the `buildSrc` root dir.

Reply:
it's the parent of the root project scope.

Reply:
Got it. I guess what I had in mind when commenting is that the current name is confusing to me. `buildSrcClassLoaderScope` or `rootProjectParentClassLoaderScope` would be more explicit. wdyt? &lt;/nitpick&gt;

Reply:
We now use the term `baseProjectClassLoaderScope` for this everywhere. WDYT?